tools/aequatus/aequatus.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="aequatus-generator" name="Aequatus generator" version="0.1.1">
+    <description>Aquatus generator takes tree, gene and alignment information and join them in as aequatus format.</description>
+     <requirements>
+        <requirement type="package" version="2.90">perl_json</requirement>
+    </requirements>
+    <command>
+    <![CDATA[
+python $__tool_directory__/aequatus-generator.py 
+-t $treeFile 
+-g $genesFile
+-f $source.input_selector
+#if $source.input_selector=="newick"
+    -c $alignmentFile
+#end if
+> $outputFile
```

Comment:
Add `"` quotes around `$__tool_directory__/aequatus-generator.py`, `$treeFile`, `$genesFile`, `$alignmentFile` and `$outputFile`.

These will be translated to file paths, which may contain spaces.